scripts/release-test-script-ut-core.sh

```diff
@@ -0,0 +1,304 @@
+#!/bin/bash
```

Comment:
not clear what the different is here.

Do you need two scripts, I'm releasing ut-core with ut-control, and therefore starting from ut-core, you need to test that your proposed branch will work for ut-control, and at the same time your proposed branch for ut-core will also work as a combination together.

Then you can be assure that they both work together.

Reply:
separated out release script for different repos.